examples_tests/39.DenoiserTonemapper/main.cpp

```diff
@@ -15,6 +15,14 @@
 
 #include "CommonPushConstants.h"
 
+/*
+	Uncomment bellow define to perform
+	DDS input/output Denoiser Tonemapper's
+	data saving.
+*/
+
+#define DDSInputOutputDenoiserSave
```

Comment:
I dont need that code anymore, just needed the DDSes

Reply:
TODO

Reply:
may merge